src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public String getRandomBall() {

```

Comment:
```suggestion
    public Ball getRandomBall() {
```
This makes codes more flexible. If we need to get a string representation of a ball, we can get it at any time, but if we want a ball itself? We wouldn't be able to get it.